includes/purge-set-object.batch.inc

```diff
@@ -0,0 +1,121 @@
+<?php
+
+/**
+ * @file
+ * Batch functions for purging ingested objects via set.
+ */
+
+
+/**
+ * Batch to purge objects ingested via a batch set.
+ *
+ * @param int $set_id
+ *   Batch set ID.
+ */
+function islandora_batch_batch_purge_object_in_set($set_id) {
+  module_load_include('inc', 'islandora_batch', 'includes/db');
+  $batch = array(
+    'title' => t('Purge objects from set'),
+    'finished' => 'islandora_batch_batch_purge_object_in_set_finished',
+    'init_message' => t('Initializing...'),
+    'operations' => array(),
+    'progress_message' => t('Time elapsed: @elapsed.'),
+    'file' => drupal_get_path('module', 'islandora_batch') . '/includes/purge-set-object.batch.inc',
+  );
+
+  $batch['operations'][] = array('islandora_batch_batch_purge_object_in_set_operation', array('set_id' => $set_id));
```

Comment:
Shouldn't really be an issue, but may be misleading, having the array of parameters be an associative array... The batch framework just passes them off positionally to the callback.